airflow/dag_processing/manager.py

```diff
@@ -561,7 +561,7 @@ def _read_from_direct_scheduler_conn(self, conn: MultiprocessingConnection) -> b
             sys.exit(os.EX_OK)
 
         self.log.debug("Received %s signal from DagFileProcessorAgent", agent_signal)
-        if isinstance(agent_signal, CallbackRequest):
+        if isinstance(agent_signal, (TaskCallbackRequest, DagCallbackRequest)):
```

Comment:
This should still work.

```suggestion
        if isinstance(agent_signal, CallbackRequest):
```

Reply:
Unfortunately, from 3.10 only 😞  https://peps.python.org/pep-0604/#isinstance-and-issubclass

Reply:
😞